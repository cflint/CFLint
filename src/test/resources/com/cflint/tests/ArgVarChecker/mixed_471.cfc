<cfcomponent displayname="mycomponent">
    <cffunction name="foo" returntype="void">
    <cfargument name="myvar" type="string" required="true">
        <cfscript>
            var id = arguments.myvar;
            var name = myvar;
            var id2 = arguments.myvar;
        </cfscript>
    </cffunction>
</cfcomponent>