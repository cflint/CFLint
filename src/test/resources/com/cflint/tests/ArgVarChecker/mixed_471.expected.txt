{
  "version" : "",
  "timestamp" : 1568522069,
  "issues" : [ {
    "severity" : "INFO",
    "id" : "ARG_VAR_MIXED",
    "message" : "ARG_VAR_MIXED",
    "category" : "CFLINT",
    "abbrev" : "AV",
    "locations" : [ {
      "file" : "src\\test\\resources\\com\\cflint\\tests\\ArgVarChecker\\mixed_471.cfc",
      "fileName" : "mixed_471.cfc",
      "function" : "foo",
      "offset" : 226,
      "column" : 24,
      "line" : 6,
      "message" : "Variable myvar should not be referenced in local and argument scope.",
      "variable" : "myvar",
      "expression" : "myvar"
    } ]
  } ],
  "counts" : {
    "totalFiles" : 0,
    "totalLines" : 0,
    "countByCode" : [ {
      "code" : "ARG_VAR_MIXED",
      "count" : 1
    } ],
    "countBySeverity" : [ {
      "severity" : "INFO",
      "count" : 1
    } ]
  }
}